templates/4b_normalize_image_data.ipynb

```diff
@@ -37,7 +37,8 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    tuning_curve_file = 'P9_450_norm_func.json'
</pre><p>What's the difference between the two Angelo Lab curves? Why is it defaulted to one over the other? And should we maybe make the default in the notebook be the ionpath P9_2600_norm_func file instead?&nbsp;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/361/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Definitely agree with making the default 2600.

Reply:
I've added a tidbit to the documentation telling the user to refer to the power supply settings to see which curve is most applicable.